### DISCUSSION TITLE
Why is my webvault saying it is on version 2.25.1 (and not current, 2.28.1)?

### DISCUSSION THREAD

**Discussion by freekvh** [dani-garcia/vaultwarden#2463] [Q&A] [answered]
I have vaultwarden/server:latest in my docker compose file, yet my webvault reports (on the login page) version 2.25.1, and not the current version of 2.28.1.

Is there something wrong? Is this to be expected?

Highest regards,

Freek

**Comment by BlackDex** [comment, marked as answer]
Because your are using the `latest` tagged image and not the `testing` tagged image.

**Reply by freekvh** [reply]
> Because your are using the `latest` tagged image and not the `testing` tagged image.

Ah, that makes sense, but 2.25.1 is recommended and secure, right? I got a bit nervous when I saw "dani-garcia: Update web vault to v2.28.1" on the main branch in this repo (not testing or development) and mine was some versions behind...

Perhaps I'm just being a noob wrt development models.

**Reply by BlackDex** [reply]
No security issues. Just some bug fixes and improvements

**Reply by freekvh** [reply]
Ok, this explains it in more detail indeed: https://github.com/dani-garcia/vaultwarden/issues/1180 (under "Which tag to use")

Thanx for your fast responses. With PW managers, one can be a bit nervous sometimes I guess.
